firebase-perf/CHANGELOG.md

```diff
@@ -23,6 +23,49 @@ Refer [GMaven](https://maven.google.com/web/index.html?q=firebase-perf#com.googl
 
 ## Released
 
+## v20.0.1 (M97)
+
+#### Android library
+
+*   {{feature}} Added Firebase console URL logging to logcat.
```

Comment:
Firebase performance logs now contain URL to the see the performance metrics on the Firebase Console.

Reply:
Done.